sdk/FEATURES.md

```diff
@@ -93,7 +96,14 @@ with Tekton's [podTemplate](https://github.com/tektoncd/pipeline/blob/master/doc
 PipelineRun. The [imagepullsecrets](/sdk/python/tests/compiler/testdata/imagepullsecrets.py) python test is an example of how to use this
 feature.
 
-This feature is recently implemented in Tekton and is available on Tekton v0.13.0 onwards.
+This feature has been available since Tekton version `0.13.0`.
+
+### Exit Handler
+
+An exit handler is a component that always executes, irrespective of success or failure, at the end of the pipeline.
```

Comment:
Can you add a pointer to the finally test case similar to the rest of the feature sections?

Reply:
I added that and capitalized the headings :-)